src/_data/sidenav.yml

```diff
@@ -106,7 +106,7 @@
           children:
             - title: Actions & shortcuts
               permalink: /development/ui/advanced/actions_and_shortcuts
-            - title: Keyboard focus
+            - title: Keyboard focus system
```

Comment:
Is this title going to be too wide for the sidebar?

Reply:
It should be ok. It does wrap text.

Reply:
Glad to see you are totally doing your holiday in style =)

Reply:
LOL. What can I say? Typical! But, I remember you did something similar not so long ago.  🗡️ 

Honestly, I suspect the baby has been born but he's in no rush to tell us.  ;)